README.md

```diff
@@ -3,125 +3,125 @@
+The steps required are outlined below.
+
+Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+
+Step 2: Clone this repository to your local machine
 
-If installing templates from a NuGet feed, the template may specify any additional components required to instantiate it (the "generator" is the most common thing to require). The `template_feed` directory in the source contains several pre-build template packs that will automatically install the required generator (in this case, it's `Microsoft.TemplateEngine.Orchestrator.RunnableProjects` which gets placed in the `feed` directory during setup).
+Step 3:  Run the setup script
+ - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
+ - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
 
-If installing templates from a ZIP or from a directory and a required component to make them is not already installed, run
+When running the setup script, the existing built in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
+files with the latest Template Engine.
 
-    dotnet new3 --install-component {the ID of the NuGet package containing the required component}
+Step 4: That's it! Now you can run `dotnet new3`.
 
-#Basic Commands
-##Showing help
+For example, here is the result of running `dotnet new3 --help` on a Mac (_truncated to save space here_).
+
+```bash
+$ dotnet new3 --help
+Template Instantiation Commands for .NET Core CLI.
+
+Usage: dotnet new3 [arguments] [options]
+
+Arguments:
+  template  The template to instantiate.
+
+<truncated>
+```
+
+# Installing templates
+
+Templates can be installed from packages in any NuGet feed, directories on the file system or ZIP type archives (zip, nupkg, vsix, etc.)
+To install a new template use the command:
+
+    dotnet new3 -i {the path to the folder containing the templates}
+
+# Basic Commands
+## Showing help
 
     dotnet new3 --help
     dotnet new3 -h
     dotnet new3
 
-##Listing templates
+## Listing templates
 
     dotnet new3 --list
     dotnet new3 -l
     dotnet new3 mvc -l            Lists all templates containing the text "mvc"
 
-##Template parameter help
+## Template parameter help
 
-    dotnet new3 MvcWebTemplate --help
-    dotnet new3 MvcWebTemplate -h
+    dotnet new3 mvc --help
+    dotnet new3 mvc -h
 
-##Template creation
+## Template creation
 
-    dotnet new3 MvcWebTemplate --name MyProject --directory --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
+    dotnet new3 MvcWebTemplate --name MyProject --output src --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
     dotnet new3 MvcWebTemplate -n MyProject -d --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
 
```

Comment:
yep, this used to refer to "create directory", it has now been broken in to a combination of a template configuration setting for preferring having a dedicated directory, the value of the name parameter (-n|--name) & the value of the output parameter (-o|--output)